src/unix/core.c

```diff
@@ -198,6 +201,19 @@ void uv__make_close_pending(uv_handle_t* handle) {
   handle->loop->closing_handles = handle;
 }
 
+int uv__getiovmax() {
```

Comment:
Sorry, I somehow overlooked that you already did that.  Can you still update the prototype though? :-)